client/app/js/crypto/module.js

```diff
@@ -83,28 +87,27 @@ angular.module('GLBrowserCrypto', [])
 // public keys. The attached-model attribute will be bound to the input of the
 // text area.
 .directive('pgpPubkeyInput', function() {
-
   return {
-    restrict: 'E',
+    restrict: 'A',
     templateUrl: '/views/partials/pgp/pubkey_input.html',
     scope: {
       localModel: '=attachedModel',
       keyForm: '=attachedForm',
+      required: '&attachedRequired',
```

Comment:
I thought a bit about this and I am cool with this solution. There is definitely a general solution in this case, I think there is a good way to handle arbitrary validators attached to a directive like this. I just don't quite know what it is.